src/components/bid_card/BidCard.js

```diff
@@ -3,14 +3,26 @@ import { dateFormatter } from 'utilities/date';
 import { convertPenniesToDollars } from 'utilities/price';
 import './BidCard.scss';
 
-const BidCard = ({ timestamp, bid_price, userName, white }) => (
-  <section className={`bid--container ${white && 'bid--white'}`}>
-    <div className='bid--container-info'>
-      <h2>{userName} Bid</h2>
-      <h5>{dateFormatter(timestamp)}</h5>
-    </div>
-    <h1 className='bid--container-price'>${convertPenniesToDollars(bid_price)}</h1>
-  </section>
-)
+const BidCard = ({ timestamp, bid_price, userName, white, winner }) => {
+  const bidClass = () => {
+    if (white && !winner) {
```

Comment:
Is white the default color? Why do you need a white prop?

Reply:
theres 3 colors:
- blue (default) (users bid page)
- white (Item bid page)
- green (winning bid)